src/librados/librados.cc

```diff
@@ -2299,7 +2299,9 @@ int librados::Rados::osd_command(int osdid, std::string cmd, const bufferlist& i
 {
   vector<string> cmdvec;
   cmdvec.push_back(cmd);
-  return client->osd_command(osdid, cmdvec, inbl, outbl, outs);
+  client->osd_command(osdid, cmdvec, inbl, outbl, outs);
+
+  return 0;
```

Comment:
i concur with @liewegas 

@jcsp this breaks this test:
```
2017-03-08T15:57:53.322 INFO:tasks.workunit.client.0.smithi156.stdout:                  api_cmd: [ RUN      ] LibRadosCmd.OSDCmdPP
2017-03-08T15:57:53.323 INFO:tasks.workunit.client.0.smithi156.stdout:                  api_cmd: /build/ceph-12.0.0-1141-gc7d986e/src/test/librados/cmd.cc:146: Failure
2017-03-08T15:57:53.323 INFO:tasks.workunit.client.0.smithi156.stdout:                  api_cmd: Value of: r == -22 || r == -ENXIO
2017-03-08T15:57:53.323 INFO:tasks.workunit.client.0.smithi156.stdout:                  api_cmd:   Actual: false
2017-03-08T15:57:53.323 INFO:tasks.workunit.client.0.smithi156.stdout:                  api_cmd: Expected: true
2017-03-08T15:57:53.323 INFO:tasks.workunit.client.0.smithi156.stdout:                  api_cmd: [  FAILED  ] LibRadosCmd.OSDCmdPP (15 ms)
```
see http://qa-proxy.ceph.com/teuthology/yuriw-2017-03-08_15:50:57-rados-wip-yuri-testing_2017_3_8---basic-smithi/895796/teuthology.log

@yuriw